public_officials/templates/registration/login.html

```diff
@@ -0,0 +1,12 @@
+
+{% block content %}
+  <h2>Login</h2>
+  <form method="post">
+    {% csrf_token %}
+    {{ form.as_p }}
```

Comment:
I think it'd be great if we only had Login with twitter instead of also signing up with user/password so we don't have to manage the security.